### PR TITLE
Add paging tests for filtering on counter columns for CASSANDRA-11629

### PR DESCRIPTION
**Pull request by ifesdjeen**

### CODE REVIEW

File: paging_test.py [hunk 1]
```diff
@@ -1085,9 +1085,71 @@ def test_paging_with_filtering(self):
             self.assertEqual(res, [[1, 1],
                                    [2, 2]])
 
+    def _test_paging_with_filtering_on_counter_columns(self, session, with_compact_storage):
+        if with_compact_storage:
+            self.create_ks(session, 'test_flt_counter_columns_copmact_storage', 2)
```
Comment (ptnapoleon) [inline paging_test.py]: nit: `copmact`.

**Comment by ptnapoleon** [comment]
+1 after nit

File: paging_test.py (hunk 1 above)
Comment (ifesdjeen) [inline paging_test.py]: thanks for catching!

**Comment by ifesdjeen** [comment]
Nit is fixed, [#11629](https://issues.apache.org/jira/browse/CASSANDRA-11629) is commited as (c08eea)[https://github.com/apache/cassandra/commit/c08eeafdcfe4b5d0fb1a8aa34ca5df27be7934ce]

**Pull request merged**